src/account-balance/account-balance.spec.js

```diff
@@ -27,6 +28,8 @@ describe('account balance monitoring', () => {
 
       // Initialize the Handler
       await (provideInitialize(initializeData))();
+
+      ({ everestId } = initializeData);
```

Comment:
Had to look this one up (parentheses around an assignment).  Nice.

Reply:
yup, need em to keep the old scope :)